.github/workflows/release.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     strategy:
       matrix:
-        python-versions: [3.11]
+        python-versions: [3.11, 3.12, 3.13]
```

Comment:
this is the release workflow, we can only run this one time, otherweise well end up with 1 release and 3 failures